### ISSUE TITLE
ReactJS app not responsive when using randomstring.generate()

### ISSUE THREAD

**Issue by bingchuan93** [klughammer/node-randomstring#37]
Hihi!

I am using your package in my ReactJS project and I realised when randomstring.generate() is called, somehow the site just hangs and becomes unresponsive. Importing the package had no issues, only when running it. Any idea why?

It can be replicated by creating a new react app, installing randomstring dependency and calling randomstring.generate() anywhere.

**Comment by JohnDDuncanIII** [comment]
I ran into the same issue after upgrading from webpack 4 to 5.

![Screen Shot 2022-10-11 at 2 51 48 PM](https://user-images.githubusercontent.com/9257785/195175643-6b4668f2-19c9-4d30-a67f-3686ba2f8ab9.png)

I fixed it by adding

```jsx
new webpack.ProvidePlugin({
    Buffer: ["buffer", "Buffer"],
})
```

to our `webpack.config.js`

See 
- https://github.com/webpack/changelog-v5/issues/10#issuecomment-615877593
- https://stackoverflow.com/a/68723223/6201291

**Comment by eliaskg** [comment]
Thanks guys. Fixed via #38
